src/elements/content-sidebar/activity-feed/task-new/__tests__/__snapshots__/TaskDueDate.test.js.snap

```diff
@@ -6,23 +6,35 @@ exports[`elements/content-sidebar/ActivityFeed/task-new/TaskDueDate should rende
   status="NOT_STARTED"
 >
   <div
-    className="bcs-TaskDueDate bcs-is-taskOverdue"
+    className="bcs-TaskDueDate"
     data-testid="task-due-date"
   >
-    <FormattedMessage
-      defaultMessage="Due: {date}"
-      id="be.contentSidebar.activityFeed.taskNew.taskDueDateLabel"
-      values={
-        Object {
-          "date": <ReadableTime
-            alwaysShowTime={true}
-            timestamp={987654321000}
-          />,
-        }
-      }
+    <LabelPill
+      type="error"
```

Comment:
We should test for the right state of the label Pill. in this case, that when overdue, we have a `LabelPill` instance with the proper `type` value (we can drill down from the wrapper via that earlier testID)